### ISSUE TITLE
[MapData] - IconScale attribute and global overrides

### ISSUE THREAD

**Issue by kristofbolyai** [Wynntils/Wynntils#2580]
I think it's beneficial to have iconScale. We used to have a lot of configuration options for vairous icon sizes, and I think, with this simple attribute, we could keep all of those, plus more.

As for the global override, I have less stable ideas. I think we should have it. I also think that if it's anything, it's not something that changes the provider loading order, rather, a simple, "manual" override (implemented by MapDataService with special care to it). I am also not sure if it should be able to override all attributes, but that could be limited by an UI.

What do you think, @magicus?

**Comment by kristofbolyai** [comment]
I just found a crucial thing we need global overrides for. We need to be able to add category and feature overrides, for many features to work:
- Clicking on waypoints to be marked
- TerritoryArea color overrides for resource generation/guild color
- Changing only a single map features settings, i.e. user wants to make the Detlas blacksmith iconScale bigger.
- Providing category overrides this way (for user settings)

Basically the idea is to be to not have any explicit "marked state" for a feature, rather, provide a way for us to apply overrides to features, that make them marked. That way, the feature itself can provide the color, and other settings for it's marker, but an override can enable/disable the feature being marked.

**Comment by kristofbolyai** [comment]
IconScale is a config of WorldMarkersFeature now. Global overrides are added. I'll close this now.
